clang/unittests/Format/FormatTest.cpp

```diff
@@ -22858,6 +22858,31 @@ TEST_F(FormatTest, FormatsLambdas) {
       "      //\n"
       "    });");
 
+  verifyFormat("int main() {\n"
```

Comment:
Get yourself a style with a lower column limit `auto Style = getLLVMStyleWithColumnLimit(40);` and use that, so that the lines don't have to be wrapped here, that's easier to read.

Reply:
Cache the style and re use it.